.mailmap

```diff
@@ -1620,6 +1621,7 @@ cym1 <16437732+cym1@users.noreply.github.com>
 damianos <damianos@semmle.com>
 dandiez <47832466+dandiez@users.noreply.github.com>
 der-blaue-elefant <github@kklein.de>
+digkioan <125358066+digkioan@users.noreply.github.com>
```

Comment:
This line is not needed and should not be here.

Reply:
Sorry, I forgot to remove it. Thank you. I will change it immediately.